tests/e2e/test_run_module_output.py

```diff
@@ -26,7 +27,7 @@ async def test_run_module_output():
     host = "127.0.0.1"
     endpoint = "/api/ws2"
     proc_id = "42"
-    port = 31416
+    port = utils.get_safe_port()
```

Comment:
Is this on purpose, or are we missing a parameter here?

Reply:
@zkayyali812 Sorry by the confusion. I noticed that my previous approach was wrong so I've changed it. Now is not needed a parameter, since the port is chosen by the helper. 
Please, review again.